Ghidra/Features/Base/src/main/java/ghidra/program/util/SymbolicPropagator.java

```diff
@@ -39,7 +39,7 @@
 import ghidra.util.exception.*;
 import ghidra.util.task.TaskMonitor;
 
-public class SymbolicPropogator {
+public class SymbolicPropagator {
```

Comment:
This change here will *probably* get rejected. I *think* I recall seeing a pr previously that addressed this one but was deemed not worth it.

If you stick to typos in documentation and error messages, things the user sees, it should be ok.

Reply:
Agreed, we don't want to break the API, at least not right now.  Like @astrelsky said, if you make this PR just about Strings we should be able to take it.

Reply:
Agreed, we don't want to break the API, at least not right now. Like @astrelsky said, if you make this PR just about Strings we should be able to take it.